test/integration/tests/checkout/happy.js

```diff
@@ -189,4 +189,36 @@ describe(`paypal checkout component happy path`, () => {
             });
         });
     });
+
+    it('should render checkout with default dimensions', (done) => {
+        const DEFAULT_POPUP_SIZE = {
+            WIDTH:  500,
+            HEIGHT: 590
+        };
+
+        runOnClick(() => {
+            let childWindow;
+            onWindowOpen().then(win => {
```

Comment:
The goal is to assert on the width and height of the popup. Currently it seems like `onWindowOpen` along with `done()` will solve it. But I'm definitely open to other suggestions. I tried using Promises but couldn't get it to error from the `onWindowOpen` callback.